server/v2/cometbft/handlers/defaults.go

```diff
@@ -153,18 +146,36 @@ func (h *DefaultProposalHandler[T]) ProcessHandler() ProcessHandler[T] {
 	}
 }
 
+// decodeTxs decodes the txs bytes into a decoded txs
+// If there a fail decoding tx, remove from the list
+// TODO: should return an err here?
+func decodeTxs[T transaction.Tx](codec transaction.Codec[T], txsBz [][]byte) []T {
+	var txs []T
+	for _, tx := range txsBz {
+		decTx, err := codec.Decode(tx)
+		if err != nil {
+			// TODO: vote extension meta data as a custom type to avoid possibly accepting invalid txs
```

Comment:
i think it's important to understand what to do here, is it fine to continue with txs if they fail decoding?

Reply:
yes agree, I think we should decide about it. Like v1 we reject all if have a tx decoded fail.